Traceback (most recent call last):
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\ibhan\anaconda3\envs\mfdoc39\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
bau_2100 =baseline.loc[2100,'PAKCCEMISCO2TKN']
bau_2022 = baseline.loc[2022,'PAKCCEMISCO2TKN']
bau_growth_rate = (bau_2100/bau_2022)**(1/(2100-2022))

target_2100 = baseline.loc[2100,'PAKCCEMISCO2TKN']*(1-reduction_percent/100)
target_growth_rate = (target_value/bau_2022)**(1/(2100-2022))

print(f"Business as usual Emission value in 2100: {BAU_2100:13,.0f}")
print(f"Business as usual Emission value in 2100: {target_2100:13,.0f}")
print(f"Business as usual growth rate in percent: {bau_growth_rate-1:13,.1%}")
print(f"Target growth rate in percent           : {target_growth_rate-1:13,.1%}")
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Cell [1;32mIn [9], line 6[0m
[0;32m      3[0m bau_growth_rate [38;5;241m=[39m (bau_2100[38;5;241m/[39mbau_2022)[38;5;241m*[39m[38;5;241m*[39m([38;5;241m1[39m[38;5;241m/[39m([38;5;241m2100[39m[38;5;241m-[39m[38;5;241m2022[39m))
[0;32m      5[0m target_2100 [38;5;241m=[39m baseline[38;5;241m.[39mloc[[38;5;241m2100[39m,[38;5;124m'[39m[38;5;124mPAKCCEMISCO2TKN[39m[38;5;124m'[39m][38;5;241m*[39m([38;5;241m1[39m[38;5;241m-[39mreduction_percent[38;5;241m/[39m[38;5;241m100[39m)
[1;32m----> 6[0m target_growth_rate [38;5;241m=[39m (target_value[38;5;241m/[39mbau_2022)[38;5;241m*[39m[38;5;241m*[39m([38;5;241m1[39m[38;5;241m/[39m([38;5;241m2100[39m[38;5;241m-[39m[38;5;241m2022[39m))
[0;32m      8[0m [38;5;28mprint[39m([38;5;124mf[39m[38;5;124m"[39m[38;5;124mBusiness as usual Emission value in 2100: [39m[38;5;132;01m{[39;00mBAU_2100[38;5;132;01m:[39;00m[38;5;124m13,.0f[39m[38;5;132;01m}[39;00m[38;5;124m"[39m)
[0;32m      9[0m [38;5;28mprint[39m([38;5;124mf[39m[38;5;124m"[39m[38;5;124mBusiness as usual Emission value in 2100: [39m[38;5;132;01m{[39;00mtarget_2100[38;5;132;01m:[39;00m[38;5;124m13,.0f[39m[38;5;132;01m}[39;00m[38;5;124m"[39m)

[1;31mNameError[0m: name 'target_value' is not defined
NameError: name 'target_value' is not defined

